src/types.ts

```diff
@@ -0,0 +1,5 @@
+import type { APIMessage } from "discord-api-types/v10";
+
+export interface DiscordMessage extends APIMessage {
```

Comment:
This isn't really discord specific imo - optimistic should simply be a feature to allow us to dim the message to show that it is in a sending state

Reply:
I'm not sure what you mean.